app/controllers/reservations_controller.rb

```diff
@@ -249,7 +249,7 @@ def checkout # rubocop:disable all
       r = Reservation.find(r_id)
       checked_out_reservations <<
         r.checkout(r_attrs[:equipment_object_id], current_user,
-                   Hash.new(r_attrs[:checkout_procedures]), r_attrs[:notes])
```

Comment:
I'm wondering why we changed the `checkin` method call but not this one, weird...